### ISSUE TITLE
jss.removeStyleSheet method

### ISSUE THREAD

**Issue by nathanmarks** [cssinjs/jss#294]
It would be useful to have a `removeStyleSheet` method that accepted a reference to a `StyleSheet` object did the following:

1) Detach stylesheet from the DOM
2) Remove from sheet registry

Right now I have to manually splice the registry array to remove the sheet reference from the registry. If it is part of the public API it won't be prone to breaking should `SheetRegistry` internals change.

**Comment by kof** [comment]
What about this?

```javascript
sheet.detach()
jss.registry.remove(sheet)
```

**Comment by kof** [comment]
However probably `jss.removeStyleSheet()` also makes sense as opposite to `jss.createStylesheet()`

**Comment by kof** [comment]
Will implement now quickly this low hanging fruit.

**Comment by kof** [comment]
done, will be released in the next release
